Bump `@wordpress/scripts@17.1.0` from `12.5.0`

tomalec opened the pull request:
(_take 3 on updating `wp-scripts`_)
:warning: This PR requires #1133 to see if bundle size does not increase too much.

### Changes proposed in this Pull Request:

- :nail_care: Whitespace alignment in php file, 
	to make `lint:php` happy.
- Update `@wordpress/scripts@17.1.0` from `12.5.0`, 
   Add `postcss@8.3.5` as this is a required peer dependency.
   Update webpack conf to use the latest version of Sass options.
   
   The idea is to reduce the lag and technical debt we have with our dependencies.


### Screenshots:

<!--- Optional --->
Bundle size seems unaffected
![image](https://user-images.githubusercontent.com/17435/144451825-8055ddfe-3b38-4bf5-b6eb-73223e675f69.png)



### Detailed test instructions:

1. checkout this branch
2. `npm install && npm run build && npm run test:js`
3. `npm test:e2e` currently fials due to https://github.com/woocommerce/google-listings-and-ads/issues/1129




<!--
Optional.
Enter a summary of all changes in this Pull Request, which will be added to the changelog if accepted.
Each line should start with `(Fix|Add|Tweak|Update) - `, for example:
> Fix - I took care of something that wasn't working.
> Add - I added something new that's pretty cool.
> Tweak - I made a small change.
> Update - I made big changes to something that wasn't broken.

Leave the "Changelog entry" header in place completely empty, without any summary if no changelog entry is needed.
If you remove the "Changelog entry" header, the title of Pull Request will be used as the changelog entry.
-->
### Changelog entry

> Tweak - Updated `@wordpress/scripts@17.1.0`.

tomalec committed:
Update `@wordpress/scripts@17.1.0` from `12.5.0`,

Add `postcss@8.3.5` as this is a required peer dependency.
Update webpack conf to use the latest version of Sass options.

tomalec committed:
Update package-lock.json

tomalec committed:
Whitespace alignment in php file,

to make `lint:php` happy.

eason9487 reviewed (commented):
LGTM. Ran all scrips that use `wp-scripts` and all can be run. 👍 

Only one shows a deprecated warning that could be updated by the way.

```bash
$ npm run format:js

> Please note that the `format-js` script has been renamed to `format`.
> If you're calling it from any of your own scripts, please update them accordingly.
```

tomalec committed:
Use `format` instead of `format-js`,

to avoid `@wordpress/scripts` deprecation warning.

Addresses https://github.com/woocommerce/google-listings-and-ads/pull/1132#pullrequestreview-822440060

tomalec commented:
Nice catch, thanks. Renamed to `format`.
